packages/mermaid/src/mermaidAPI.ts

```diff
@@ -514,6 +517,13 @@ const render = function (
     throw e;
   }
 
+  // This is the d3 node for the svg element
```

Comment:
Good stuff. Found an issue here though as this is only catering the render flow and not the renderAsync flow. See below.